server/cluster.go

```diff
@@ -99,7 +99,10 @@ type ClusterReq struct {
 	// Cluster is desynchronized.
 	Signature string
 
+	// Client message. Set for C2S requests.
 	Pkt *ClientComMessage
```

Comment:
Maybe name them a bit more contextually, i.e. `CliMsg` and `SrvMsg` or some such?

Reply:
Sure.